lib/doorkeeper-sequel/mixins/application_mixin.rb

```diff
@@ -70,29 +72,11 @@ def by_uid(uid)
         first(uid: uid.to_s)
       end
 
-      def supports_confidentiality?
-        column_names.include?('confidential')
-      end
-
       def column_names
         columns.map(&:to_s)
       end
     end
 
-    # Fallback to existing, default behaviour of assuming all apps to be
-    # confidential if the migration hasn't been run
-    def confidential
```

Comment:
@nbulaj, Is there a specific reason for removing this and above methods? I updated my gemfile and i got

`NameError: undefined method `confidential' for class `Doorkeeper::Application'`

at startup.

Reply:
Oh man, No. This might be the reason for #16. Let me test.

Reply:
Yes I did created. This error is from api only project. (Wrong alarm). I still have #16.